pkg/modules/aws/analyze/security_group_analysis.go

```diff
@@ -0,0 +1,42 @@
+package analyze
+
+import (
+	"github.com/praetorian-inc/janus-framework/pkg/chain"
+	"github.com/praetorian-inc/janus-framework/pkg/chain/cfg"
+	"github.com/praetorian-inc/nebula/internal/registry"
+	"github.com/praetorian-inc/nebula/pkg/links/aws"
+	"github.com/praetorian-inc/nebula/pkg/links/options"
+	"github.com/praetorian-inc/nebula/pkg/outputters"
+)
+
+func init() {
+	registry.Register("aws", "analyze", AWSSecurityGroupAnalysis.Metadata().Properties()["id"].(string), *AWSSecurityGroupAnalysis)
+}
+
+var AWSSecurityGroupAnalysis = chain.NewModule(
+	cfg.NewMetadata(
+		"AWS Security Group Analysis",
+		"Analyzes security groups to find related ENIs, VPCs, subnets, and AWS assets.",
```

Comment:
```suggestion
		"AWS Security Group Analysis",
		"Correlates network resources to security groups.",
```

I don't love the suggestion, but I think the original description is too vague, and I didn't understand what it did until I reviewed the output. It's a really useful module and I want to make sure folks don't gloss over it when they see the description